app/routes/entries.tsx

```diff
@@ -1,7 +1,25 @@
 import { Outlet } from "@remix-run/react"
+import { type ActionArgs } from "@remix-run/server-runtime"
 import Header from "~/components/elements/Header"
 import Main from "~/components/elements/Main"
 import Nav from "~/components/elements/Nav"
+import { updateRecordByAttributeAndType } from "~/models/entry.server"
+import {
+  getNumberFromFormInput,
+  getStringFromFormInput,
+} from "~/utils/generalUtils"
+
+export async function action({ request }: ActionArgs) {
```

Comment:
Move to entries/headword route so user stays on the same page.

Reply:
also done-- this was a bit stranger, cause there were problems with the redirect etc. unfortunately. So there's a bit of a workaround in the code